docs/01-command-line-reference.md

```diff
@@ -268,6 +269,47 @@ The flags listed below are only available via the command line interface. All ot
 
 Print the help document.
 
+#### `-p <plugin>`, `--plugin <plugin>`
+
+Use the specified plugin. There are several ways to specify plugins here:
+
+- Via a relative path:
+  
+  ```
+  rollup -i input.js -f es -p ./my-plugin.js
+  ```
+  
+  The file should export a plugin object or a function returning such an object.
+- Via the name of a plugin that is installed in a local or global `node_modules` folder:
+  
+  ```
+  rollup -i input.js -f es -p @rollup/plugin-node-resolve
+  ```
+  
+  If the plugin name does not start with `rollup-plugin-` or `@rollup/plugin-`, Rollup will automatically try adding these prefixes:
+  
+  ```
+  rollup -i input.js -f es -p node-resolve
+  ```
+
+- Via an inline implementation:
+
+  ```
+  rollup -i input.js -f es -p '{transform: c => "/* TEST */" + c}
```

Comment:
typo - need a trailing single quote

Reply:
Looks like the example doesn't work unless you add a newline - rollup drops the comment otherwise:
```
-p '{transform: c => "/* TEST */\n" + c}'
```

Reply:
This example is a bit more interesting - it's up to you:
```
-p '{transform: (c, i) => `/* ${i} */\n` + c}'
```

Reply:
Or perhaps:
```
-p '{transform: (c, i) => `/* ${JSON.stringify(i)} */\n` + c}'
```
which would show embedded `\0` (nils) for other plugins.

Reply:
Nice catch. Without the newline, Rollup only keeps the comment if the first line is preserved, which was the case for my test call 😉 With the newline, it is always preserved as it is considered a banner.

I will change to using your example 👍

Reply:
Fixed. I also consistenly used single quotes around both examples because otherwise, Bash might do variable expansions or let hell break loose when you use backticks...